tests/helpers/test_entityfilter.py

```diff
@@ -17,31 +21,43 @@ def test_no_filters_case_1():
 def test_includes_only_case_2():
```

Comment:
We should not alter the behavior of existing tests beyond that is needed to make the function not reject argument changes, it is better to add new ones so we prove the old tests are still working without changes.

Reply:
Ok going to go back through carefully with an eye on this.  Just did the ones in this test suite, going to go through the others as well.

Reply:
Reverted changes to tests where I added glob filters and added separate ones specifically for that.